controllers/v1/questionsController.js

```diff
@@ -118,6 +118,12 @@ module.exports = class Questions extends Abstract {
           name : solutionDocument.entityType
         },{ profileFields : 1 }).lean();
 
+        if (!entityTypeDocument) {
```

Comment:
@deepakr123 - Let's do it only for survey

Reply:
resolved